tests/background/uri-info-test.js

```diff
@@ -61,39 +33,90 @@ describe('background/uri-info', () => {
       'chrome://newtab',
       'chrome-extension://fadpmhkjbfijelnpfnjmnghgokbppplf/pdfjs/web/viewer.html?file=http%3A%2F%2Fwww.pdf995.com%2Fsamples%2Fpdf.pdf',
       'file://whatever',
-    ].forEach(badURI => {
-      it('does not send request to API if URI does not have an allowed protocol', async () => {
-        assert.strictEqual(await uriInfo.getAnnotationCount(badURI), 0);
-        assert.equal(fetchStub.callCount, 0);
+    ].forEach(blockedProtocol => {
+      it('throws for blocked protocol', () => {
+        try {
+          uriInfo.shouldQueryUri(blockedProtocol);
+        } catch (error) {
+          assert.equal(error.message, 'Blocked protocol');
+        }
       });
     });
 
-    ['https://www.google.com', 'http://www.example.com'].forEach(okURI => {
-      it('sends request to API if URI does not match blocklist entries and has an allowed protocol', async () => {
-        assert.strictEqual(await uriInfo.getAnnotationCount(okURI), 1);
-        assert.equal(fetchStub.callCount, 1);
+    ['https://www.google.com/', 'http://www.example.com/'].forEach(okURI => {
+      it('sends request to API if URI does not match blocklist entries and has an allowed protocol', () => {
```

Comment:
This test description needs updating as the function being tested (`shouldQueryUri`) doesn't "send a request to the API" but instead "returns a normalized URI".